src/resspect/fit_lightcurves.py

```diff
@@ -130,14 +122,14 @@ def fit_snpcc(
         File names prefix
      number_of_processors: int, default 1
         Number of cpu processes to use.
-     feature_extractor: str, default bazin
+     feature_extractor: str, default Bazin
         Function used for feature extraction.
     """
-    if feature_extractor == 'bazin':
+    if feature_extractor == 'Bazin':
```

Comment:
This feels like an opportunity for a common interface, but it sounds like that may be out of scope for this PR